pwiz_tools/Shared/Common/Chemistry/MzTolerance.cs

```diff
@@ -60,16 +60,30 @@ public MzTolerance(double value = 0, Units units = Units.mz)
             return 0;
         }
 
+        public double GetMzTolerance(double mz)
+        {
+            switch (Unit)
+            {
+                case Units.mz:
+                    return Value;
+                case Units.ppm:
+                    return Math.Abs(mz) * Value * 1e-6;
+            }
+
+            return 0;
+
+        }
+
         /// <summary>returns true iff a is in (b-tolerance, b+tolerance)</summary>
-        public static bool IsWithinTolerance(double a, double b, MzTolerance tolerance)
+        public bool IsWithinTolerance(double a, double b)
```

Comment:
I have a question. Why the inequality is non-inclusive here while it is inclusive for the method match tolerance?

Reply:
Good question.
@chambm, is there a reason that this method is using an exclusive range?
I see that this C# method is not used anywhere but there is a C++ method in "MZTolerance.cpp" which has the same implementation.
Skyline always (as far as I know) interprets tolerances using an inclusive range so that if someone sets the tolerance to zero there will be one number which would still fall in the tolerance range.